css/style.css

```diff
@@ -0,0 +1,113 @@
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+.search-div {
+    padding: 10px;
+    border-top: 1px #ced4da solid;
+}
+
+.search-div a {
+    text-decoration: none;
+}
+
+.search-div:first-child {
+    border-top: none;
+}
+
+.container-flex {
+    display: flex;
+}
+
+.container-flex.header {
+    margin-top: 40px;
+    margin-bottom: 20px;
+}
+
+.header-img {
+    width: 60px;
+    height: 60px;
+}
+
+.container-flex img {
+    /* width: 40px;
+    height: 40px; */
+    width: 100%;
+    height: 100%;
+}
+
+.container-flex a {
+    font-size: 28px;
+    margin-left: 30px;
+    text-decoration: none;
+}
+
+.container-flex p {
+    font-size: 18px;
+}
+
+.container-flex #company-percentages {
+    margin-left: 10px;
+}
+
+.myChart {
+    margin-top: 20px;
+    width: 80vw;
+}
+
+.image-icone {
+    width: 30px;
+    height: 30px;
+    margin-right: 10px;
+}
+
+.marquee {
+    background: linear-gradient(-135deg, #9FCC9F, #74AB8D);
+    width: 98vw;
+    box-sizing: border-box;
+    padding: 1em;
+    color: white;
+    font-weight: 200;
+    display: flex;
+    align-items: center;
+    overflow: hidden;
+    margin: 0 auto;
+    padding: 0;
+    height: 10vh;
+}
+
+.marquee-content {
+    display: flex;
```

Comment:
when using the same css in couple of different places it is better to have a class for it like so:

flex-box {
   display: flex;
}